R/modify_pd.R

```diff
@@ -0,0 +1,53 @@
+#' Update indention information of parse data
+#'
+#' @param pd A nested or flat parse table that is already enhanced with
+#'   line break and space information via [create_filler()] or
+#'   [create_filler_nested()].
+#' @param indent_by How many spaces should be added after the token of interest.
+#' @name update_indention
+NULL
+
+#' @rdname update_indention
+indent_round <- function(pd, indent_by) {
+  if (any(pd$token == "')'")) {
+    start <- 2
```

Comment:
see above, I think we need to explicitly find the start token because of the two scenarios.